packages/mask/src/extension/popups/components/PopupLayout/index.tsx

```diff
@@ -0,0 +1,66 @@
+// ! This file is used during SSR. DO NOT import new files that does not work in SSR
+
+import { memo, type PropsWithChildren } from 'react'
+import { GlobalStyles, Paper } from '@mui/material'
+import { makeStyles } from '@masknet/theme'
+import { Outlet, matchPath, useLocation } from 'react-router-dom'
+import { Navigator } from '../Navigator/index.js'
+import { PopupRoutes } from '@masknet/shared-base'
+
+const GlobalCss = (
+    <GlobalStyles
+        styles={{
+            body: {
+                minWidth: 400,
+                overflowX: 'hidden',
+                margin: '0 auto !important',
+                maxWidth: '100%',
+                WebkitFontSmoothing: 'subpixel-antialiased',
+                '&::-webkit-scrollbar': {
+                    display: 'none',
+                },
+            },
+        }}
+    />
+)
+
+const useStyles = makeStyles()({
+    container: {
+        height: '100%',
+        minHeight: 560,
```

Comment:
The `minHeight` for the popup in the new design draft is 620.

Reply:
OK, would set to 600 which is max height of popup limited by Chrome.
https://developer.chrome.com/docs/extensions/reference/browserAction/#popup